tools/speech_data_explorer/data_explorer.py

```diff
@@ -207,10 +205,13 @@ def load_data(data_filename, disable_caching=False, estimate_audio=False, vocab=
                     num_chars = 1e-9
                 data[-1]['WER'] = round(word_dist / num_words * 100.0, 2)
                 data[-1]['CER'] = round(char_dist / num_chars * 100.0, 2)
-                data[-1]['WMR'] = round(num_matches / num_words * 100.0, 2)
+                data[-1]['WMR'] = round(measures['hits'] / num_words * 100.0, 2)
+                data[-1]['I'] = measures['insertions']
+                data[-1]['D'] = measures['deletions']
+                data[-1]['D-I'] = measures['deletions'] - measures['insertions']
```

Comment:
Maybe rename D to `D (Deletion)` in the table form, same for I? Since the abbreviation wasn't introduced in the Statistics tab, unlike CER, WER?